config.py

```diff
@@ -51,6 +52,7 @@
     "aws": ("aws", "api", "ebs", "ec2", "efs", "emr", "rds", "ml", "mq", "nat", "vpc", "waf", "sdk"),
     "azure": ("ad", "b2c", "ai", "api", "cdn", "ddos", "dns", "fxt", "hana", "hd", "id", "sap", "sql", "vm"),
     "gcp": ("gcp", "ai", "api", "cdn", "dns", "gke", "gpu", "iap", "ml", "nat", "os", "sdk", "sql", "tpu", "vpn"),
+    "firebase": ("fcm", ),
```

Comment:
Last request: `Ab` of ABTesting and `Ml` of MLKit should be uppercase. Could you please add these in this tuple? (maybe `ab`, `fcm`, `ml`)

Reply:
Just as you said, the class name is `ABTesting`, `MLKit` is correct.
I fixed it, and re-run `autogen.sh`